SQLQueryGroupProjectTeamB1Julia.sql

```diff
@@ -0,0 +1,255 @@
+--Julia Shull
+--Group Project Code
+--November 2019
+
+USE Proj_B1
+
+--STORED PROCEDURES

```

Comment:
please nest the insert statements of your stored procedures into explicit transaction statements